src/notification.h

```diff
@@ -12,6 +12,8 @@
 #define NORM 1
 #define CRIT 2
 
+#define MAX_CHARS 5000
```

Comment:
Not really, but I do agree that it's too generic maybe `NOTIFICATION_MAX_LENGTH`?